content/pt/docs/concepts/configuration/pod-overhead.md

```diff
@@ -0,0 +1,197 @@
+---
+reviewers:
+- dchen1107
+- egernst
+- tallclair
+title: Pod Overhead
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Quando executa um Pod num nó, o próprio Pod usa uma quantidade de recursos do sistema. Estes
+recursos são adicionais aos recursos necessários para executar o(s) _container(s)_ dentro do Pod.
+Sobrecarga de Pod, do inglês _Pod Overhead_, é uma funcionalidade que serve para contabilizar os recursos consumidos pela
+infraestrutura do Pod para além das solicitações e limites do _container_.
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+No Kubernetes, a sobrecarga de _Pods_ é definido no tempo de
+[admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+de acordo com a sobrecarga associada à
+[RuntimeClass](/docs/concepts/containers/runtime-class/) do _Pod_.
+
+Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma dos 
+pedidos de recursos do _container_ aquando do agendamento de um Pod. Semelhantemente, o _kubelet_
+vai incluir a sobrecarga do Pod aquando do dimensionamento do cgroup do Pod, e aquando de
+levar a cabo a classificação de despejo do Pod.
+
+## Possibilitando a Sobrecarga do Pod {#set-up}
+
+Terá de garantir que o [portão de funcionalidade](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodOverhead` está ativo (está ativo por defeito a partir da versão 1.18)
```

Comment:
default = padrão
```suggestion
`PodOverhead` está ativo (está ativado por padrão a partir da versão 1.18)
```

Reply:
Here "by default" is correctly translated to "por defeito" and "ativo" is correct both in pt_pt and pt br. Do you acknowledge this?

Reply:
ok, no problem being that way:
```suggestion
`PodOverhead` está ativo (está ativo por padrão a partir da versão 1.18)
```

Reply:
Hello @devlware it means the same, ("by default" is widely used in pt_pt as "por defeito").

Reply:
Hi @devlware and @joao-luna-98 I think we can use both, to keep close pt-BR and pt-PT.
Like this: "está ativado por padrão ou por defeito"

What you think guys?

Reply:
IMO we can use @jcjesus suggestion but "por defeito" even been widely used in pt_pt would be a little strange for pt_BR.
This is a nice question, how should we provide this compatibility between pt BR and PT?

Reply:
I think we need to create like a tag to describe "por defeito[1]" and in the end page, this number or tag or HTML <sub>[1]</sub> tag, this is very used in book or article to reference another article or book, will contain the explanation: this term is must use in Portugal, such as "por padrão" is the same in Brazil. I think this is a good solution to solve this and we are can learn about the difference between the costumes.

Reply:
Would this be added in this PR or should we open another one? As doing that would require some investigation on my part. But sounds like a great idea.